server/src/main/java/org/uiautomation/ios/wkrdp/message/AbstractApplicationSentListingMessage.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2012-2013 eBay Software Foundation and ios-driver committers
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.uiautomation.ios.wkrdp.message;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.dd.plist.NSDictionary;
+
+public abstract class AbstractApplicationSentListingMessage extends BaseIOSWebKitMessage implements
+    ApplicationSentListingMessage {
+
+  protected String applicationIdentifier;
+
+  protected List<WebkitPage> pages;
+
+  private static final Logger LOG = Logger.getLogger(AbstractApplicationSentListingMessage.class.getName());
+
+  public AbstractApplicationSentListingMessage(String rawMessage) throws Exception {
+    super(rawMessage);
+    pages = new ArrayList<>();
+    populatePages();
+    populateApplicationIdentifierKey();
+    if (LOG.isLoggable(Level.FINE))
+      LOG.fine("got: " + this);
```

Comment:
Should we be enhancing this log message to include a better message instead of just `got:` which doesn't seem to add a lot of info to help debug problems.